### DISCUSSION TITLE
[BUG] No se visualiza el botón de creación de canales en el código

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#64] [4. Bug Reports] [answered]
Hola Buenos dias

Me percaté que en el admin, sección grupos, se encuentra habilitado el botón **permitir la creación de canales** como **Activo**. Sin embargo al revisar el código no se **visualiza**. Sólo se muestra cuando se desactiva.

Adjunto imagen del admin
![image](https://user-images.githubusercontent.com/61419471/114196128-162cc200-991f-11eb-9aa0-0ca0cb0ddb1f.png)

**Adjunto código del grupo create channel de la imagen adjunta**

```
 {
    "__v" : NumberInt(36114), 
        "isActive" : true, 
        "onboarding" : true, 
        "text" : "Escribe tus preguntas en la sección Pregúntale a Buddy y recibe respuestas de manera automática, si el sistema no tiene la respuesta, pregúntasela a un experto o a tus colegas.", 
        "image" : ""
    }, 
    "createChannels" : {
        "allowedContentTypes" : [

        ], 
        "requiredPermission" : [

        ]
    }, 
    "propertyLevelsConfig" : [

    ], 
```

**Adjunto código cuando se desactiva la creación del canal**

```
"createChannels" : {
        "requiredPermission" : [

        ], 
        "requiredPermissionsV2" : [

        ], 
        "allowedContentTypes" : [

        ], 
        "active" : false, 
        "openChannel" : true
    }, 
```

Esta bién que se comporte de esa manera?

**Comment by acedward** [comment, marked as answer]
@NicolasPLG parece un bug, por mientas - para asegurar un comportamiento determinístico podríás:
*  Si lo quieres **ON**, desactivar save, luego, activar save.
*  Si lo quieres **OFF**, desactivar save.
Así aseguras que el parámetro queda setado en **true** ó **false**, pero no quede sin determinar.

@guillevalin FYI

**Reply by acedward** [reply]
Quizás solo sea de grupos antiguos.
@guillevalin quizás hacer una migración y setarlos en falso los no definidos?

**Reply by NicolasPLG** [reply]
@acedward al hacer on y save y despues off si aparece el campo (código 2). Y al pregunta salió porque quería hacer masivamente desde la bd el desactivar la creación de canal. Lo reporto o esperemos a que @guillevalin responda? :>

**Reply by acedward** [reply]
@NicolasPLG si requieres que sea corregido, por favor contactate con tu representante de Cotalker @jdurandpaz 
Voy a cerrar el ticket.
